app/views/groups/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="container">
+  <div class="splash header">
+    <% if user_signed_in? %>
+      <%= link_to destroy_user_session_path, method: :delete do %>
+        <%= image_tag("menu.png", alt: "Menu icon") %>
+      <% end %>
+    <% end %>
+    <h2>CATEGORIES</h2>
+    <%= image_tag("search.png", alt: "Search icon") %>
+  </div>
+
+  <ul class="sub-container">
+    <% @groups.each do |group| %>
+      <li class="group">
+        <%= link_to group_path(group) do %>
+          <% if group.icon.present? %>
+            <%= image_tag group.icon %>
+          <% else %>
+            <%= image_tag asset_path('under-construction.jpeg') %>
+          <% end %>
+          <p><%= group.name %></p>
+          <h2>$<%= group.movements.sum(:amount) %></h2>
+        <% end %>
+      </li>
+    <% end %>
+  </ul>
```

Comment:
![image](https://user-images.githubusercontent.com/79058364/229174252-43062fdb-5311-4d5d-bf87-bdaca7dfb9fc.png)


- [ ] As requested by the previous reviewer ⬆️ display the creation date for each category 👇🏻  

![image](https://user-images.githubusercontent.com/79058364/229173064-8a9d6048-29a7-40f7-82ac-b38e3493ed5c.png)